torch/testing/_internal/common_methods_invocations.py

```diff
@@ -8520,6 +8523,7 @@ def ref_pairwise_distance(input1, input2):
            assert_autodiffed=True,
            supports_forward_ad=True,
            supports_fwgrad_bwgrad=True,
+           expect_nnc_fusion=False,
```

Comment:
@eellison lmk if you have any general comments on this. If you think it looks good I'll go through the list of ops and disable the ones where nnc fusion currently fails with `expect_nnc_fusion=False` like here.